HelpAssistHer/help-assist-her
# Update list of services

## Description
* Update the list of services in the `pregnancy-center-services` file
* Move the `pregnancy-center-services` file into the `pregnancy-centers` directory
* Create a helper file so that both Joi and Mongoose can build their schemas based off of the `pregnancy-center-services` array, meaning we then only have to update services in one file.
* Add an "Other services" text box
* Update services in `import-google-sheets.py`
* Update services in `test-data.js`
* Run import script to generate new .json file with updated services

## Test Plan
1. Go to `http://localhost:4000`
1. Retrieve a resource with the Get One Resource button
1. You should see the list of new services as well as the Other Services text box
1. Click on some of the check boxes next to the services
1. Click Submit and make sure that the data was saved successfully
1. Go through data import steps to ensure data can be imported successfully with updated services
